localstack/utils/aws/message_forwarding.py

```diff
@@ -210,7 +218,7 @@ def list_of_parameters_to_object(items):
     return {item.get("Key"): item.get("Value") for item in items}
 
 
-def send_event_to_api_destination(target_arn, event):
+def send_event_to_api_destination(target_arn, event, http_parameters: Dict = None):
```

Comment:
```suggestion
def send_event_to_api_destination(target_arn, event, http_parameters: Optional[Dict] = None):
```

Reply:
not super important right now, just a small nit, will merge without this for now